Assignment_Week_3.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+

```

Comment:
Thanks for uploading your solution/test case! Going forward, we're also hoping to give you guys feedback on your pen/paper TEBOW IT, so make sure to upload that as well :)